mantle/platform/api/azure/instance.go

```diff
@@ -55,8 +56,12 @@ func (a *API) getVMParameters(name, userdata, sshkey, storageAccountURI string,
 	//             from the following: Contains an uppercase character, Contains a
 	//             lowercase character, Contains a numeric digit, Contains a special
 	//             character) Control characters are not allowed"
-	password := fmt.Sprintf("%s%s%s", "ABC&", strconv.Itoa(rand.Int()), "xyz")
-
+	n, err := rand.Int(rand.Reader, big.NewInt(999999999999999))
```

Comment:
This is not quite 50 bits of entropy.  That's fine, but it's an unusual number; maybe we should go for 64?

Reply:
would you use something like math.MaxInt here?

Reply:
MaxInt doesn't make since for a BigInt, but you could use `1 << 64`.